### PR TITLE
Add Dev tools

### LINKED ISSUE #61: 0.0.10 breaks devtools
I originally filed this issue on freeqaz/redux-simple-router-example#1. I am able to reproduce the behavior elsewhere by switching between `0.0.8` and `0.0.10`, so maybe this is a more appropriate place to file the issue?

Gif for your viewing pleasure:

The only change I see to the state as reported by devtools is the addition of `changeId`.

### PR DESCRIPTION
**Pull request by geowarin**
This pull request add dev-tools to the basic example.
I hope this will help to fix #61.

### CODE REVIEW

**Comment by kjbekkelund** [comment]
@geowarin I created another PR based on this one that attempts to solve #61. Check it out here: #73

File: examples/basic/webpack.config.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ module.exports = {
 // This will make the redux-simpler-router module resolve to the
 // latest src instead of using it from npm. Remove this if running
 // outside of the source.
-var src = path.join(__dirname, '..', '..', 'src')
+var src = path.join(__dirname, '..', '..', 'lib')
```
Comment (jlongster) [inline examples/basic/webpack.config.js]: Please remove this change. We want to pull the latest source, not the latest built source. We are already running it through webpack and it'll build it again.

File: examples/basic/components/App.js [hunk 2]
```diff
@@ -25,5 +25,5 @@ function App({ updatePath, children }) {
 
 module.exports = connect(
   null,
-  { updatePath }
+  { replacePath }
```
Comment (jlongster) [inline examples/basic/components/App.js]: Can you change this to `pushPath` instead?

**Comment by jlongster** [comment]
Looks great, thanks, I needed to do this soon but this helps a lot. @kjbekkelund  I may go ahead and merge this so we can keep discussing solutions.

**Comment by geowarin** [comment]
Hi @jlongster, I amended my branch according to your remarks.
@kjbekkelund, sorry I rebased and push forced so you might need to repair your branch :confounded:

**Comment by kjbekkelund** [comment]
@jlongster Absolutely — this should be merged! :)

@geowarin No prob, my branch was just to get a discussion going on how to solve the devtools problem :)

**Comment by jlongster** [comment]
Ok I'll go ahead and merge it and look into it today.

**Pull request merged**